transactions/2.0/_index.md

```diff
@@ -5,19 +5,19 @@ summary: "Release for Jakarta EE 9"
 ---
 Jakarta Transactions - Need scope statement
 
-* [Jakarta Transactions 2.0 Release Record]()
```

Comment:
A couple of requested changes earlier in this file...  Change "under development" to "milestone".  And, change the date to something current such as today's date 2020-06-11.  This date: field is used to indicate a publish date or an embargo date.  It can be used to indicate a future date so that a page won't go live until some event happens.  But, we want these (milestone) items to go live as soon as we merge them.  Thanks.

title: "Jakarta Transactions 2.0 (milestone)"
date: 2020-06-11

Reply:
This one is updated in the latest push